src/phone-number-input/phone-number-input.tsx

```diff
@@ -45,7 +46,7 @@ export const PhoneNumberInput = ({
         setInputValue(
             PhoneNumberInputHelper.formatNumber(value?.number, selectedOption)
         );
-    }, [options, value]);
+    }, [value]);
```

Comment:
i think this is sufficient since either change will trigger if you specify them separately

Reply:
with `<PhoneNumberInput value={{ number, countryCode }} />` the object gets recreated on every rerender. but not a huge deal, the parent can also do its own memoization